packages/playwright-core/src/server/firefox/ffPage.ts

```diff
@@ -533,7 +533,15 @@ export class FFPage implements PageDelegate {
   }
 
   async setInputFilePaths(handle: dom.ElementHandle<HTMLInputElement>, files: string[]): Promise<void> {
-    throw new Error('Not implemented');
+    await Promise.all([
+      this._session.send('Page.setFileInputFiles', {
+        frameId: handle._context.frame._id,
+        objectId: handle._objectId,
+        files
+      }),
+      handle.dispatchEvent('input'),
```

Comment:
Do we need `{ 'bubbles': true }` like we have in `InjectedScript.setInputFiles`?

Reply:
`ElementHandle.dispatchEvent` does it by default.